Src/TestSample.BroadInt.Tests/Users/User.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Users

```

Comment:
I believe the idea here for Broad Integration tests would be to hit each endpoint in a given service, run one positive business scenario and get an appropriate response, I would think one test per method would be sufficient. Thoughts?